docs/content/routing/entrypoints.md

```diff
@@ -737,3 +737,59 @@ entrypoints.websecure.http.tls.domains[1].sans=foo.test.com,bar.test.com
     entrypoints.websecure.address=:443
     entrypoints.websecure.http.tls.certResolver=leresolver
     ```
+
+### listen on some ip addresses only
+
+```toml tab="File (TOML)"
+[entryPoints.oneIPv4Address]
+  address = "192.168.2.7:8888" # network ip address INSIDE container
```

Comment:
> network ip address INSIDE container

Not sure this comment should be included in these examples. Traefik doesn't always run in a container. Also the word INSIDE is confusing, as many runtimes assign the IP to the container, not a device inside the container.

Reply:
I see. Maybe: ip address seen by traffic, may be different from host

Reply:
I removed the comment